Makefile

```
@@ -328,6 +328,9 @@ tear-down-kind2-env:
 test-e2e-local-fast: $(GINKGO)
 	./hack/test-e2e-local.sh --label-filter "Shoot && fast"
 
+test-e2e-local-cpm: $(GINKGO)
+	./hack/test-e2e-local.sh --label-filter "CPM"
```

Call that label `control-plane-migration` or `migration`. Acronyms hinder communication.
```suggestion
	./hack/test-e2e-local.sh --label-filter "Shoot && control-plane-migration"
```